auctions/english-auction/vercel.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
We don't need it actually since we will be deploying to gh pages instead of vercel.  I'll delete it.  But if you are curious it is just a file to configure vercel (a frontend deployment tool)